docs/source/daml/intro/8_Exceptions.rst

```diff
@@ -11,30 +11,35 @@ from an error and continue the transaction instead of aborting it.
 
 One option for doing that is to represent errors explicitly via
 ``Either`` or ``Option`` as shown in :doc:`3_Data`. This approach has
-the advantage that it is very explicit about which operations can fail
-and which cannot. However, it also has some downsides. First, it can
-be invasive for operations where aborting the transaction is often the
-desired behavior, e.g., changing division to return ``Either`` or an
-``Option`` to handle division by zero would be a very invasive change
-and many callsites might not want to handle the error case explicitly.
-Second, and more importantly, this approach does not allow rolling
-back ledger actions that have happened up to this point: If a contract
-got created before we hit the error, there is no way to undo this.
-
-Exceptions provide a way to handle certain types of errors in a
-non-invasive way and to roll back parts of the transaction that
-lead up to this error. All of that still happens within the same
-transaction and is thereby atomic contrary to handling the error
-outside of Daml.
+the advantage that it is very explicit about which operations are
```

Comment:
Can you please not reflow this in the same PR. It’s very hard to see what has actually changed.